### PR TITLE
feat: Single Project - DevExpressApp

### PR DESCRIPTION
**Pull request by vatsashah45**

### CODE REVIEW

File: UI/MauiEmbedding/DevExpressApp/DevExpressApp/ReadMe.md [hunk 1]
```diff
@@ -35,4 +35,4 @@ For more information on how to use the controls from DevExpress in an Uno Platfo
  ![DevExpress Sample App running on iOS](doc/assets/DevExpress_iOS.png)
```
Comment (eriklimakc) [inline UI/MauiEmbedding/DevExpressApp/DevExpressApp/ReadMe.md:35]: It seems like the screenshot images are not showing anymore.

**Review by eriklimakc — COMMENTED** [review]
Make sure that everything related to code is under a `src` folder and that the readme file is under the main `DevExpressApp` folder.

File: UI/MauiEmbedding/DevExpressApp/DevExpressApp/nuget.config
Comment (eriklimakc) [inline UI/MauiEmbedding/DevExpressApp/DevExpressApp/nuget.config:1]: I guess this file must be removed in order to the CI build to pass. You will then need to comment all the `DevExpress` nuget packages as it was on the "multi-project" version.

Is that correct @nickrandolph @agneszitte ?

File: UI/MauiEmbedding/DevExpressApp/DevExpressApp/Platforms/iOS/Entitlements.plist
Comment (agneszitte) [inline UI/MauiEmbedding/DevExpressApp/DevExpressApp/Platforms/iOS/Entitlements.plist:1]: We should not have MacCatalyst related files as it is iOS/Android only

**Comment by agneszitte** [comment]
Closing this PR in favor of https://github.com/unoplatform/Uno.Samples/pull/748

**Pull request closed without merging**